app/static/css/_results.scss

```diff
@@ -84,3 +87,45 @@
     list-style: none;
     padding: 0;
 }
+
+
+.show_more_button {
```

Comment:
if these are going to be styled identically (and you always want them to be), you could do it like `.show_more_button, .show_less_button {` and save the repeated CSS

Reply:
cool!